tests/params/basic.yml

```diff
@@ -391,3 +391,16 @@ classes:
     new_ag:
       enter_type: new_agent
       prob: 1.0
+  age_bins:
+    min:
+      min_age: 0
+      max_age: 15
+    0:
+      min_age: 16
+      max_age: 24
+    1:
+      min_age: 25
+      max_age: 44
+    2:
```

Comment:
why the mix of words and numbers?

Reply:
It's testing to make sure words don't break it. Since we're no longer enforcing that bins be named by number, changing to string seemed like a good idea, but maybe I should make it all strings in that case...